### PR TITLE
Implement localization

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

**Comment by andy5995** [comment]
A lot of changed files here. That's due to me removing getName() in package.*, and splitting it into getFormalName() and getBinName().

**Comment by winrid** [comment]
So why is player.name getter called getFormalName now?

**Comment by andy5995** [comment]
btw, I'm using getBinName() as an arg to some new functions called, right under the main() block.

The biggest question at this point in the patch is where to put this block:

```
#ifdef ENABLE_NLS
#include <locale.h>
#endif
#include "gettext.h"
#define _(String) gettext (String)
#define gettext_noop(String) String
#define N_(String) gettext_noop (String)
```

Any file that has some strings marked in it will need that.

**Comment by winrid** [comment]
can create a new Localization.hpp and put it there

**Comment by andy5995** [comment]
> So why is player.name getter called getFormalName now?

Ah shit, because I did oversearched and replaced. I'll fix that!

**Comment by winrid** [comment]
ok. wasn't sure if you were adding an informal name to players. :D

File: meson.build [hunk 1]
```diff
@@ -121,45 +126,42 @@ endif
...
-vorbis_dep = dependency('vorbis', required: false)
-if not vorbis_dep.found()
-  vorbis_dep = cc.find_library('vorbis', has_headers: 'vorbisfile.h', required: false)
-  if not vorbis_dep.found()
-    vorbis_dep = dependency(
-      'vorbis',
-      required: true,
-      fallback: 'vorbis',
-      default_options: ['default_library=static'])
+# These are required by the SDL2_ttf meson wrap, but not by NetPanzer
```
Comment (andy5995) [inline meson.build:146]: A bit unrelated to this patch... but when updating the appimage script, I noticed freetype being compiled in.

File: po/meson.build [hunk 2]
```diff
@@ -0,0 +1,37 @@
+dep_intl = []
+if get_option('nls')
+  inc_args = []
+  dep_intl = dependency('intl', required: false)
+  if not dep_intl.found()
```
Comment (andy5995) [inline po/meson.build:5]: I might be able to get rid of all these conditions, and get it working on Windows by using the [proxy-libintl](https://github.com/frida/proxy-libintl) wrap maintained by @frida

**Pull request merged**